ui/apps/platform/src/Containers/AccessControl/AuthProviders/ConfigurationFormFields.tsx

```diff
@@ -283,6 +283,29 @@ function ConfigurationFormFields({
                             />
                         </FormGroup>
                     </GridItem>
+                    <GridItem span={6} lg={6}>
+                        <FormGroup
+                            fieldId="config.disable_offline_access_scope"
+                            helperText={
+                                <span className="pf-u-font-size-sm">
+                                    Could be used if identity provider has limit on number of
+                                    offline tokens issued.
```

Comment:
nit: probably a question for a native speaker, but I would have said `Should` instead of `Could` 🤔 
```suggestion
                                    Shouldbe used if the identity provider has a limit on the number of
                                    offline tokens that can be issued.
```

Reply:
Aha, then what about the following? If **that can be issued** means issued by the identify provider.

Use if the identify provider has a limit on the number of offline tokens that it can issue.

Reply:
Thanks for the input, folks  - updated the helper text